tsconfig.json

```diff
@@ -17,7 +17,7 @@
       "jest",
       "node"
     ],
-    "target": "es6",
```

Comment:
Does this the change to ES2020 target affect the client-side code?  If so do the major browsers all support ES2020?